src/download.ts

```diff
@@ -42,7 +41,6 @@ export type CustomHnsDownloadOptions = CustomDownloadOptions & {
 
 export type CustomGetMetadataOptions = BaseCustomOptions & {
   endpointGetMetadata?: string;
-  query?: Record<string, unknown>;
```

Comment:
This is directly exposed to the client right? Is removing this breaking compatibility?

Reply:
Yes. We already removed `query` from other options, I just forgot about this spot.